### PR TITLE
new module: indeppairwise

### PR DESCRIPTION
**Pull request by atrigila**
## PR checklist

Closes [#3147](https://github.com/nf-core/modules/issues/3147) 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

**Review by sateeshperi — COMMENTED** [review]
lgtm. Good job!

File: modules/nf-core/plink/indeppairwise/meta.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  - bim:
+      type: file
+      description: PLINK extended MAP file
+      pattern: "*.{bim}"
+  - fam:
+      type: file
+      description: PLINK sample information file
+      pattern: "*.{fam}"
+  - window_size:
+      type: value
```
Comment (pinin4fjords) [inline modules/nf-core/plink/indeppairwise/meta.yml:37]: suggest 'number' or 'integer' as appropriate for these parameters

File: modules/nf-core/plink/indeppairwise/meta.yml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  - variant pruning
+  - bim
+  - fam
+tools:
+  - "plink":
+      description: "Whole genome association analysis toolset, designed to perform a range of basic, large-scale analyses in a computationally efficient manner."
+      homepage: "https://www.cog-genomics.org/plink"
+      documentation: "https://www.cog-genomics.org/plink/1.9/data#"
+      tool_dev_url: "https://www.cog-genomics.org/plink/1.9/dev"
+
```
Comment (pinin4fjords) [inline modules/nf-core/plink/indeppairwise/meta.yml:15]: remove the random empty line?

File: modules/nf-core/plink/indeppairwise/meta.yml [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+  - plink
+  - indep pairwise
+  - variant pruning
+  - bim
+  - fam
+tools:
+  - "plink":
+      description: "Whole genome association analysis toolset, designed to perform a range of basic, large-scale analyses in a computationally efficient manner."
+      homepage: "https://www.cog-genomics.org/plink"
+      documentation: "https://www.cog-genomics.org/plink/1.9/data#"
```
Comment (pinin4fjords) [inline modules/nf-core/plink/indeppairwise/meta.yml:13]: Probably don't need the `#`

File: tests/modules/nf-core/plink/indeppairwise/main.nf [hunk 4]
```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { PLINK_INDEPPAIRWISE } from '../../../../../modules/nf-core/plink/indeppairwise/main.nf'
+include { PLINK_VCF           } from '../../../../../modules/nf-core/plink/vcf/main.nf'
+
+workflow test_plink_indeppairwise {
+
+input = [
```
Comment (pinin4fjords) [inline tests/modules/nf-core/plink/indeppairwise/main.nf:10]: Fix the indent?

**Review by pinin4fjords — COMMENTED** [review]
I've picked a few nits on some minor formatting points, but otherwise nice tidy work

**Review by pinin4fjords — APPROVED** [review]
LGTM

**Pull request merged**